src/sandstorm/grain.capnp

```diff
@@ -401,6 +401,14 @@ interface UiView {
     # two UiViews are really the same view with different permissions applied, and can then combine
     # them in the UI as appropriate.
     #
+    # Implementation-wise, in addition to the permissions enumerated here, there is an additional
+    # "is human" pseudopermission provided by the Sandstorm platform, which is invisible to apps,
```

Comment:
"pseudoprivilege" sounds better, but I want to invoke the concept of `Permission`s like the ones declared by the `ViewInfo`, so I'd better use "pseudopermission" everywhere.